pkg/cmd/taskrun/describe.go

```diff
@@ -104,6 +104,10 @@ or
 				opts.TaskrunName = args[0]
 			}
 
+			if opts.Status {
+				return trdesc.PrintTaskRunStatus(cmd.OutOrStdout(), opts.TaskrunName, p)
```

Comment:
I think it would be nice if the task would use exit code so script can usually do this : 

i.e: 

exit 0 => Succeed
exit 1 => Failed
exit 2 = > Unknown (anything else than success or failed i.e: timeout)

There is some [conventions](https://unix.stackexchange.com/a/254747) for shell script returncodes but i don't think we need to go as complicated....

Reply:
I used `formatted.Condition` to get the text of status currently, should I define a new function to meet the requirement?
Besides, `corev1.ConditionUnknown` is defined as `Running` in package `formatted`, is there any possibility that it represents other status?

Reply:
I have tried to introduce the status code.

```suggestion
				code, err := trdesc.PrintTaskRunStatus(cmd.OutOrStdout(), opts.TaskrunName, p)
				exit(code)
```

Would the implementation like above seems weird since the rest parts handle the error with the unified error?